packages/logs/src/boot/logs.ts

```diff
@@ -53,10 +55,11 @@ export function doStartLogs(
     combine({ session_id: session.getId() }, getGlobalContext(), getRUMInternalContext())
   )
 
-  const assemble = buildAssemble(session, configuration)
+  const errorFilter = createErrorFilter(configuration, reportError)
+  const assemble = buildAssemble(session, configuration, errorFilter)
```

Comment:
nit: I would even put createErrorFilter inside buildAssemble but no strong argument for it